.env.sample

```diff
@@ -1,14 +1,14 @@
 OPENCTI_ADMIN_EMAIL=admin@opencti.io
-OPENCTI_ADMIN_PASSWORD=ChangeMe
-OPENCTI_ADMIN_TOKEN=ChangeMe
-MINIO_ROOT_USER=ChangeMeAccess
-MINIO_ROOT_PASSWORD=ChangeMeKey
+OPENCTI_ADMIN_PASSWORD=ChangeMe_STRING
+OPENCTI_ADMIN_TOKEN=ChangeMe_UUIDv4
+MINIO_ROOT_USER=ChangeMeAccess_STRING
+MINIO_ROOT_PASSWORD=ChangeMeKey_STRING
 RABBITMQ_DEFAULT_USER=guest
 RABBITMQ_DEFAULT_PASS=guest
-CONNECTOR_HISTORY_ID=ChangeMe
-CONNECTOR_EXPORT_FILE_STIX_ID=ChangeMe
-CONNECTOR_EXPORT_FILE_CSV_ID=ChangeMe
-CONNECTOR_EXPORT_FILE_TXT_ID=ChangeMe
-CONNECTOR_IMPORT_FILE_STIX_ID=ChangeMe
-CONNECTOR_IMPORT_DOCUMENT_ID=ChangeMe
+CONNECTOR_HISTORY_ID=ChangeMe_UUIDv4
+CONNECTOR_EXPORT_FILE_STIX_ID=ChangeMe_UUIDv4
+CONNECTOR_EXPORT_FILE_CSV_ID=ChangeMe_UUIDv4
+CONNECTOR_EXPORT_FILE_TXT_ID=ChangeMe_UUIDv4
+CONNECTOR_IMPORT_FILE_STIX_ID=ChangeMe_UUIDv4
+CONNECTOR_IMPORT_DOCUMENT_ID=ChangeMe_UUIDv4
```

Comment:
Added type `STRING` and `UUIDv4` here so that it's clear without having to refer back to the documentation. This also allowed removal within the README.

Reply:
Good idea, thanks